static/src/questionnaires/QuestionnaireMetadataCreate.vue

```diff
@@ -8,32 +8,31 @@
     <form @submit.prevent="createMetadata">
       <fieldset class="form-fieldset">
         <div class="form-group">
-          <label class="form-label">Titre<span class="form-required">*</span></label>
-          <input type="text" class="form-control" v-model="metadata.title" required>
-          <span class="text-muted" v-if="metadata.title">
-            Le titre sera affiché comme ceci :
-            <strong>Questionnaire n°{{ questionnaireNumbering }} - {{ metadata.title }}</strong>
+          <label class="form-label">Quel titre souhaitez vous donner au questionnaire n°{{ questionnaireNumbering }} ?</label>
```

Comment:
Pour l'accessibilité, j'ai commencé a mettre les trucs de Aria, qui sert pour les screenreaders par exemple. 

Ici il faut idealement in `aria-labelledby` sur le `input` : voir Exemple 1 ici :
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute#Examples

Et `aria-describedby` pour le texte d'explication : https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute

Reply:
J'ai rajouté. Mais je n'étais pas très sûr. Disons que se sera à revoir ça si on fait une revue plus globale de l’accessibilité.